scripts/provisioning/build-prep-1node.sh

```diff
@@ -1,31 +1,72 @@
 #!/bin/sh
 
 # Clone, build and prepare to start a single-node cluster.
+#
+# Usage: ./build-prep-1node.sh [-dev]
+#
+#   -dev  development mode, don't build and install rpms
+#
+set -e -o pipefail
 
-set -eu -o pipefail
+dev_mode=
```

Comment:
Add comments like:
# set this to "YES" to build source code in local dir and then install only motr services
# and leave it empty to build RPMs and install them.

Reply:
Hmm, isn't it enough what's written in the "Usage" above - "development mode, don't build and install rpms"?